src/Parser/TypeParser.php

```diff
@@ -503,29 +503,32 @@ private function tryParseArrayOrOffsetAccess(TokenIterator $tokens, Ast\Type\Typ
 	private function parseArrayShape(TokenIterator $tokens, Ast\Type\TypeNode $type): Ast\Type\ArrayShapeNode
 	{
 		$tokens->consumeTokenType(Lexer::TOKEN_OPEN_CURLY_BRACKET);
-		if ($tokens->tryConsumeTokenType(Lexer::TOKEN_CLOSE_CURLY_BRACKET)) {
-			return new Ast\Type\ArrayShapeNode([]);
-		}
 
-		$tokens->tryConsumeTokenType(Lexer::TOKEN_PHPDOC_EOL);
-		$items = [$this->parseArrayShapeItem($tokens)];
+		$items = [];
+		$sealed = true;
```

Comment:
@jrmajor this cause error on Rector side, if this goes to `false` for default value, it may reduce the possible error.

that's sealed true make all changes to add `...` which actually not, see

https://github.com/rectorphp/rector-src/actions/runs/3706328433/jobs/6281403238

we forced pin to use 1.5.0 and the issue disappear

/cc @TomasVotruba @ondrejmirtes